src/MDMMotionCurve.h

```diff
@@ -144,6 +144,6 @@ typedef NS_ENUM(NSUInteger, MDMSpringMotionCurveDataIndex) {
 /**
  Timing information for an iOS modal presentation slide animation.
  */
-#define MDMModalMovementTiming { \
+#define MDMModalMovementTiming (MDMMotionTiming){ \
```

Comment:
I've never seen this before. What do the parentheses do for a macro? Can you link to documentation about its use?

Reply:
When assigning C structs to local variables, the following works:

```objc
MDMMotionTiming timing = {
  .delay = 0.000, .duration = 0.500, .curve = _MDMSpring(3, 1000, 500)
};
```

but the following gives an error of "Expected expression":

```objc
MDMMotionTiming timing;
timing = {
  .delay = 0.000, .duration = 0.500, .curve = _MDMSpring(3, 1000, 500)
};
```

I'm assuming this is because the Objective-C compiler isn't able to infer what type the `{ }` is meant to be. Doing the following resolves the error though:

```objc
MDMMotionTiming timing;
timing = (MDMMotionTiming){
  .delay = 0.000, .duration = 0.500, .curve = _MDMSpring(3, 1000, 500)
};
```

Thus the change to this macro. I've pulled the change out to a separate PR though so that we can focus on the unit test in this PR.

Reply:
Got it! So it's basically an inline typecast?

Reply:
Yep!